test/functional/varint.js

```diff
@@ -80,42 +97,171 @@ describe('Varint tables', function() {
+                limit: 3,
+                attributes: {
+                    key: 'testing',
+                    rev: { lt: 2 }
                 }
-            });
+            }
         })
         .then(function(result) {
             deepEqual(result.body.items.length, 1);
+            deepEqual(result.body.items[0].key, 'testing');
+            deepEqual(result.body.items[0].rev, 1);
+        });
+    });
+
+    it('retrieves using gt predicate', function() {
+        return router.request({
+            uri: '/restbase.cassandra.test.local/sys/table/varintTable/',
+            method: 'get',
+            body: {
+                table: 'varintTable',
+                limit: 3,
+                attributes: {
+                    key: 'testing',
+                    rev: { gt: 1 }
+                }
+            }
         })
-        .then(function () {
-            return router.request({
-                uri: '/restbase.cassandra.test.local/sys/table/varintTable/',
-                method: 'get',
-                body: {
-                    table: 'varintTable',
-                    limit: 3,
-                    attributes: {
-                        key: 'testing',
-                        rev: { ge: 1 }
-                    }
+        .then(function(result) {
+            deepEqual(result.body.items.length, 1);
+            deepEqual(result.body.items[0].key, 'testing');
+            deepEqual(result.body.items[0].rev, 5);
+        });
+    });
+
+    it('retrieves using le predicate', function() {
+        return router.request({
+            uri: '/restbase.cassandra.test.local/sys/table/varintTable/',
+            method: 'get',
+            body: {
+                table: 'varintTable',
+                limit: 3,
+                attributes: {
+                    key: 'testing',
+                    rev: { le: 5 }
                 }
-            });
+            }
+        })
+        .then(function(result) {
+            deepEqual(result.body.items.length, 2);
+        });
+    });
+
+    it('retrieves using ge predicate', function() {
+        return router.request({
+            uri: '/restbase.cassandra.test.local/sys/table/varintTable/',
+            method: 'get',
+            body: {
+                table: 'varintTable',
+                limit: 3,
+                attributes: {
+                    key: 'testing',
+                    rev: { ge: 1 }
+                }
+            }
         })
         .then(function(result) {
             deepEqual(result.body.items.length, 2);
         });
     });
+
+    it('retrieves using multiple predicates', function() {
+        return router.request({
+            uri: '/restbase.cassandra.test.local/sys/table/varintTable/',
+            method: 'get',
+            body: {
+                table: 'varintTable',
+                limit: 3,
+                attributes: {
+                    key: 'testing',
+                    rev: { eq: 1 },
```

Comment:
IMO, we can leave it (as a less-known option or the like) on the basis of `eq` being a valid operator (as it should be), as well as being consistent with the usage of other operators (`lt`, `gt`, *etc*). However, I think we should definitely encourage the usage of direct values, instead of wrapping them in `{ eq: X }` objects.